packages/core/src/components/popover/popover.tsx

```diff
@@ -134,31 +134,11 @@ export interface IPopoverProps extends IOverlayableProps, IProps {
      */
     popoverClassName?: string;
 
```

Comment:
this is a major API deprecation. What is the upgrade path for existing code?

Reply:
Note: this interface inherits the new lifecycle interface, so this will just be a rename.

Reply:
#2587